server/src/main/scala/controllers/AdminController.scala

```diff
@@ -65,3 +39,62 @@ class AdminController @Inject() (
     } yield Ok(Json.toJson(maskedResponse))
   }
 }
+
+object AdminController {
+  import sttp.model.StatusCode
+  import sttp.tapir.*
+  import sttp.tapir.json.play.*
+
+  private val getUserLogsEndpoint = endpoint.get
+    .in("admin" / "users" / path[UUID]("userId") / "logs")
```

Comment:
This is repeating what we have in the routes file unnecessarily

Reply:
Added a TODO for next PR